js/core/dom_component.js

```diff
@@ -12,7 +12,7 @@ import { grep, noop } from './utils/common';
 import { inArray } from './utils/array';
 import { isString, isDefined } from './utils/type';
 import { hasWindow } from '../core/utils/window';
-import { resize as resizeEvent, visibility as visibilityEvents } from '../events/core/events_engine';
+import { resize as resizeEvent, visibility as visibilityEvents } from '../events/index';
```

Comment:
I did not get why this new module named as `index`. We have a lot of modules which are semanticly connected with the `event` word so it becomes a mess. I'd like to know more about this module need for by looking at it's name.